File: problems/algorithms/function-area-of-circle/js/response/area-of-circle.test.js
import {
  areaOfCircle
} from './area-of-circle.js'

describe('Circle Tools', () => {

  test('Area of the circle of radius 10 is of 31.41592653589793', () => {
    expect(areaOfCircle(10)).toBe(314.1592653589793)
  })

  test('Area of the circle of radius 1 is of 3.141592653589793', () => {
    expect(areaOfCircle(1)).toBe(3.141592653589793)
  })

})

File: problems/algorithms/function-area-of-circle/python/response/area_of_circle.test.py
import unittest
from area_of_circle import areaOfCircle

class Sum(unittest.TestCase):
  "Circle Tools"

  def test_should_calculate_area_of_circle(self):
    "Area of the circle of radius 10 is of 31.41592653589793"
    self.assertEqual(areaOfCircle(10), 314.1592653589793)
    self.assertEqual(areaOfCircle(1), 3.141592653589793)

if __name__ == '__main__':
    unittest.main()